### PR TITLE
slight rephrasing to improve flow

### PR DESCRIPTION
**Pull request by Daniel-Mietchen**

### CODE REVIEW

File: Policies/github-bug-bounty-program-legal-safe-harbor.md [hunk 1]
```diff
@@ -31,4 +31,4 @@ That said, if legal action is initiated by a third party, including law enforcem
 
 ### 3. Limited Waiver of Other Site Polices
 
-To the extent your security research activities are inconsistent with certain restrictions in our [relevant site policies](/categories/site-policy/) but are consistent with the terms of our bug bounty program, we waive those restrictions for the sole and limited purpose of permitting your security research under this bug bounty program. Just like above, if in doubt, ask us first!
```
Comment (SaavyGirl) [inline Policies/github-bug-bounty-program-legal-safe-harbor.md:34]:
```suggestion
To the extent your security research activities are inconsistent with certain restrictions in our [relevant site policies](/categories/site-policy/) but are consistent with the terms of our bug bounty program, we waive those restrictions for the sole and limited purpose of permitting your security research under this bug bounty**[[]()]()** program. Just like above, if in doubt, ask us first!
```

File: Policies/github-bug-bounty-program-legal-safe-harbor.md [hunk 2]
```diff
@@ -31,4 +31,4 @@ That said, if legal action is initiated by a third party, including law enforcem
 
 ### 3. Limited Waiver of Other Site Polices
```
Comment (SaavyGirl) [inline Policies/github-bug-bounty-program-legal-safe-harbor.md:32]: - @Daniel-Mietchen I'm really trying idk what I'm doing Idk how I even set this coding thing up awhile back.

File: Policies/github-bug-bounty-program-legal-safe-harbor.md [hunk 3]
```diff
@@ -31,4 +31,4 @@ That said, if legal action is initiated by a third party, including law enforcem
...
 
-To the extent your security research activities are inconsistent with certain restrictions in our [relevant site policies](/categories/site-policy/) but are consistent with the terms of our bug bounty program, we waive those restrictions for the sole and limited purpose of permitting your security research under this bug bounty program. Just like above, if in doubt, ask us first!
+To the extent that your security research activities are inconsistent with certain restrictions in our [relevant site policies](/categories/site-policy/) but consistent with the terms of our bug bounty program, we waive those restrictions for the sole and limited purpose of permitting your security research under this bug bounty program. Just like above, if in doubt, ask us first!
```
Comment (SaavyGirl) [inline Policies/github-bug-bounty-program-legal-safe-harbor.md:34]: opening up the source code
```suggestion
To the extent that your security research activities are inconsistent with certain restrictions in our [relevant site policies](/categories/site-policy/) but consistent with the terms of our bug bounty program, we waive those restrictions for the sole and limited purpose of permitting your security research under this bug bounty program. Just like above, if in doubt, ask us first!
```

File: Policies/github-bug-bounty-program-legal-safe-harbor.md (hunk 1 above)
Comment (SaavyGirl) [reply Policies/github-bug-bounty-program-legal-safe-harbor.md:34]: I'm trying to learn this and fix everything that I need to accordingly.  Thank you all for your patience with me.

Comment (SaavyGirl) [inline Policies/github-bug-bounty-program-legal-safe-harbor.md:34]: Thankyou for help

**Review by SaavyGirl — CHANGES_REQUESTED** [review]

File: Policies/github-bug-bounty-program-legal-safe-harbor.md (hunk 3 above)
Comment (mlinksva) [reply Policies/github-bug-bounty-program-legal-safe-harbor.md:34]: Hi @Daniel-Mietchen thanks for this suggestion. I can't discern whether either flow is better, but am inclined to merge this unless someone better at grammar or editing has a different take. cc @vollmera

**Review by mlinksva — APPROVED** [review]

**Review by 122oa — APPROVED** [review]

**Review by SaavyGirl — APPROVED** [review]

File: Policies/github-bug-bounty-program-legal-safe-harbor.md (hunk 3 above)
Comment (SaavyGirl) [reply Policies/github-bug-bounty-program-legal-safe-harbor.md:34]: > opening up the source code
> ```suggestion
> To the extent that your security research activities are inconsistent with certain restrictions in our [relevant site policies](/categories/site-policy/) but consistent with the terms of our bug bounty program, we waive those restrictions for the sole and limited purpose of permitting your security research under this bug bounty program. Just like above, if in doubt, ask us first!
> ```
> 

Shared

Comment (SaavyGirl) [reply Policies/github-bug-bounty-program-legal-safe-harbor.md:34]: - Thankyou[**__**]()

Comment (Cu2y) [inline Policies/github-bug-bounty-program-legal-safe-harbor.md:34]: great job!

**Review by Jayson29-undroid — COMMENTED** [review]
n

**Review by Cu2y — COMMENTED** [review]
great improvement flow.

File: Policies/github-bug-bounty-program-legal-safe-harbor.md (hunk 2 above)
Comment (SaavyGirl) [reply Policies/github-bug-bounty-program-legal-safe-harbor.md:32]: Looks good

**Pull request merged**